modules/study/src/main/NewPgnImport.scala

```diff
@@ -53,13 +61,14 @@ object NewPgnImport:
                 statusText = lila.game.StatusText(status, game.winnerColor, game.variant)
               )
             }
-            if root.tree.map(_.lastMainlineNode).exists(_.value.metas.comments.value.nonEmpty) then root
-            else
-              end.map(PgnImport.endComment).fold(root) { comment =>
-                root
-                  .focus(_.tree.some)
-                  .modify(_.modifyLastMainlineNode(_.focus(_.value.metas.comments).modify(_ + comment)))
-              }
+            val commented =
```

Comment:
this is not used anywhere, should it replace `root` on line 73

Reply:
fixed